json-schemas/interface-methods/range-filter.json

```diff
@@ -0,0 +1,55 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://identity.foundation/dwn/json-schemas/range-filter.json",
+  "type": "object",
+  "minProperties": 1,
+  "additionalProperties": false,
+  "properties": {
+    "gt": {
+      "oneOf": [
+        { "type": "string" },
+        { "type": "number" }
+      ]
+    },
+    "gte": {
+      "oneOf": [
+        { "type": "string" },
+        { "type": "number" }
+      ]
+    },
+    "lt": {
+      "oneOf": [
+        { "type": "string" },
+        { "type": "number" }
+      ]
+    },
+    "lte": {
+      "oneOf": [
+        { "type": "string" },
+        { "type": "number" }
+      ]
+    }
+  },
```

Comment:
I am unsure that the newly allowed keywords `gt` and `lte` actually works, since we don't have test for them...

So either we:
- add tests to make sure they work; or 
- I'm okay if we just remove support for `gt` and `lte` at in this schema, calling this schema `date-range-filter.json`.

Obviously the latter option is a quicker way out, but I am okay with that. Though keep in mind when/if you work on the `dataSize` filter, we'd want to support all 4 keywords since it is "just" a number.